packages/toolkit/scripts/build/copyFiles.ts

```diff
@@ -94,6 +94,12 @@ const tasks: CopyTask[] = [
         target: path.join('../../node_modules', 'web-tree-sitter', 'tree-sitter.wasm'),
         destination: path.join('src', 'tree-sitter.wasm'),
     },
+
+    // AccessAnalyzer
```

Comment:
nit: worth elaborating it to indicate that it is used for PolicyChecks for Terraform templates